exporter/datasetexporter/README.md

```diff
@@ -24,6 +24,24 @@ See the [Getting Started](https://app.scalyr.com/help/getting-started) guide.
 
 If you do not want to specify `api_key` in the file, you can use the [builtin functionality](https://opentelemetry.io/docs/collector/configuration/#configuration-environment-variables) and use `api_key: ${env:DATASET_API_KEY}`.
 
+### Server Host Settings
+
+Specifying the server host is crucial for ensuring the correct functionality of DataSet. 
+DataSet expects the server host value to be provided in the `serverHost` attribute. 
+If the server host value is stored in a different attribute, you can use the [resourceprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourceprocessor/README.md) or [attributesprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/attributesprocessor) to copy it into the `serverHost` attribute.
+
+You can also utilize the `server_host` settings (described below) to populate the serverHost attribute with different values.
+
+The process of populating the serverHost attribute works as follows:
+
+* If the `serverHost` attribute is specified and not empty in the log or trace, then it is used.
```

Comment:
Do we want to clarify here (first bullet point) and be more explicit that we mean as part of the `ptrace.LogRecord` or `ptrace.Trace` attributes?

Reply:
From my perspective as user of the open telemetry I do not care about the internal classes. I think in logs and traces. If they are using SDK, those classes are called spans and traces - https://opentelemetry.io/docs/instrumentation/python/manual/ or https://opentelemetry.io/docs/instrumentation/java/